tools/src/wyvern/tools/tests/EffectSystemTests.java

```diff
@@ -0,0 +1,161 @@
+package wyvern.tools.tests;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+import wyvern.target.corewyvernIL.support.Util;
+import wyvern.tools.imports.extensions.WyvernResolver;
+import wyvern.tools.parsing.coreparser.ParseException;
+import wyvern.tools.tests.suites.RegressionTests;
+
+//import wyvern.target.corewyvernIL.support.Util;
+//import wyvern.tools.parsing.coreparser.ParseException;
+//import wyvern.tools.tests.suites.RegressionTests;
+
+/**
+ * Test suite for the effect system (adapted from ExampleTests.java).
+ * Test cases numbered with "x0" (ex. 10) do not have effect annotations; 
+ * those numbered with "xn" (ex. 12) are versions of "x0" test cases 
+ * annotated with effects. 
+ * 
+ * Successful test cases have the following printout format:
+ * "data sent: Network%d%d with(out) effects
+ * data received"
+ * 
+ * Test cases that fail due to unimplemented features beyond the parsing 
```

Comment:
We use:

    @Test
    @Category(CurrentlyBroken.class)

to mark test cases that are currently broken, instead of swallowing and printing an exception.  Either approach works in principle but better if you can make your code conform to the standard we're already using.